ERROR 24-04-23 08:55:57[test-task-scheduling3] [TaskUtils$LoggingErrorHandler:95] - Unexpected error occurred in scheduled task
java.lang.Error: Unresolved compilation problem: 
	The method EmptyTockenlt() is undefined for the type WebClientApp

	at gc.apiClient.controller.ControllerCenter.RefreshToken(ControllerCenter.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
ERROR 24-04-23 08:56:42[boundedElastic-2] [ControllerUCRM:154] - DataIntegrityViolationException 발생 : Record with the given composite key already exists.
ERROR 24-04-23 08:56:42[boundedElastic-2] [ControllerUCRM:156] - flag is 'D', delete record
ERROR 24-04-23 08:56:43[boundedElastic-2] [ServiceWebClient:183] - 57428876 has been deleted well.
ERROR 24-04-23 09:45:20[boundedElastic-2] [ControllerCenter:211] - Error Messge : Error: {"message":"A Campaign with id sortby could not be found","code":"resource.not.found","status":404,"messageWithParams":"A {entity} with id {id} could not be found","messageParams":{"id":"sortby","entity":"Campaign"},"contextId":"12302abc-9c42-419a-98f6-70b4760d97ce","details":[],"errors":[]}
ERROR 24-04-23 09:48:11[boundedElastic-2] [ControllerUCRM:154] - DataIntegrityViolationException 발생 : Record with the given composite key already exists.
ERROR 24-04-23 09:48:11[boundedElastic-2] [ControllerUCRM:156] - flag is 'D', delete record
ERROR 24-04-23 09:48:11[boundedElastic-2] [ControllerUCRM:154] - DataIntegrityViolationException 발생 : Record with the given composite key already exists.
ERROR 24-04-23 09:48:11[boundedElastic-2] [ControllerUCRM:156] - flag is 'D', delete record
ERROR 24-04-23 09:48:11[boundedElastic-2] [ServiceWebClient:183] - 57428880, 57428882 has been deleted well.
